src/index.html

```diff
@@ -2,16 +2,101 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="#">
+      <!-- Personal information -->
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input type="text" autocomplete="off" name="surname">
+        </label>
+        <label>
+          Name:
+          <input type="text" autocomplete="off" name="name">
+        </label>
+        <label>
+          How old are You?
+          <input name="old" 
+          type="number" 
+          pattern="\d+" 
+          min="1" max="100" value="12"
+        >
```

Comment:
```suggestion
          <input
            name="old" 
            type="number" 
            pattern="\d+" 
            min="1"
            max="100" 
            value="12"
          >
```
Форматування - це важливо. Кожен атрибут має бути на новому рядку з правильним рівнем відступів. 
Почитати можна тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation